dataset_interfaces/interface.py

```diff
@@ -213,8 +213,8 @@ def default_waits(self):
         assert len(self.waits) == 0 or len(self.waits) == len(self.is_question), "Current waits should be empty or the same length as the script"
         memory_span = self.dataset_generator.memory_span
 
-        # TODO: All this token counting is based on OpenAIs methods, which is not Anthropics - but Aanthopic has no reliable tokeniser released so....
-        script_tokens = get_tokens_for_script(self.script)
+        # This is done at generation time, so we pick the least efficient encoder
```

Comment:
I was confused by the wording. In this case, I guess that "less efficient" means that it uses more tokens? I'd put that instead, and why it is a better default.